src/Components/formView.tsx

```diff
@@ -289,81 +305,184 @@ export class FormView extends React.Component<Props, FormState> {
     tableColumn: ITableColumn<ICherryPickTarget>,
     tableItem: ICherryPickTarget
   ): JSX.Element => {
-    return (
-      <SimpleTableCell
-        columnIndex={columnIndex}
-        tableColumn={tableColumn}
-        key={"col-" + columnIndex}
-        contentClassName="fontWeightSemiBold font-weight-semibold fontSizeM font-size-m scroll-hidden cherry-pick-container"
-      >
-        <table className="cherry-pick-table">
-          <tbody>
-            <tr className="cherry-pick-row">
-              <td className="cherry-pick-text">Target branch:</td>
-              <td className="cherry-pick-row">
-                <div className="scroll-hidden" key={tableItem.id}>
-                  <Tooltip overflowOnly={true} key={tableItem.id}>
-                    <Observer selection={tableItem.selection}>
-                      {() => {
-                        return (
-                          <Dropdown
-                            key={tableItem.id}
-                            items={this.itemProviderGrouped}
-                            loading={this.state.loading}
-                            placeholder="Select a branch"
-                            filteredNoResultsText="No matching branch names found."
-                            onSelect={(e, newValue) =>
-                              this.handleDropdownChange(newValue, tableItem.id)
-                            }
-                            renderSelectedItems={(selection, items) =>
-                              this.retrevePreviousDropdown(
-                                selection,
-                                items,
-                                tableItem
-                              )
-                            }
-                            selection={tableItem.selection}
-                          />
-                        );
-                      }}
-                    </Observer>
-                  </Tooltip>
-                </div>
-              </td>
-            </tr>
-
-            <tr className="cherry-pick-row">
-              <td className="cherry-pick-text">Topic branch:</td>
-              <td className="cherry-pick-row">
-                <FormItem
-                  message={tableItem.errorMessage}
-                  error={tableItem.error}
-                >
-                  <TextField
-                    value={tableItem.topicBranch}
-                    onChange={(e, newValue) =>
-                      this.handleInputTopicText(newValue, tableItem.id)
-                    }
-                    placeholder="type here..."
-                    width={TextFieldWidth.auto}
+    if (tableItem.createPr) {
```

Comment:
Can we have the `if (tableItem.createPr)` block just around the `tr` entry for the pr title field instead? This would make maintaining this `SimpleTableCell` easier.